src/bptl/activiti/models.py

```diff
@@ -0,0 +1,65 @@
+from urllib.parse import urljoin
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from solo.models import SingletonModel
+
+from bptl.tasks.models import BaseTask
+
+
+class ActivitiConfig(SingletonModel):
+    """ configuration of Activiti service, including base url and credentials """
+
+    root_url = models.URLField(
+        _("camunda root"),
+        help_text=_(
+            "Root URL where activiti is installed. The REST api "
+            "path is appended to this."
+        ),
+        default="https://activiti.utrechtproeftuin.nl/activiti-app/api/",
+    )
+    rest_api_path = models.CharField(
```

Comment:
this field needs to be dropped I think, since there is no singular `rest_api_path`. In the Postman collection there are:

* `management/engine`
* `repository/process-definitions`
* `enterprise/process-definitions`
* `runtime/process-instances`

What _would_ be useful, is a field to indicate enterprise/community edition, as some API paths differ based on that (we're developing against enterprise now).

Reply:
`rest_api_path` field is removed, boolean field `enterprise` is added